### PR TITLE
Change go get command to download only

### PR DESCRIPTION
**Pull request by wpsmithtwc**
`go get github.com/akamai/cli` will automatically install the command `cli` into `$GOPATH/bin`. To avoid this you need the `-d` flag which will download only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ If you use (or want to use) [docker](http://docker.com), you can get a fully ins
...
-The container contains Akamai CLI, as well as the `purge` and `property` subcommands pre-installed.  
+The container contains Akamai CLI, as well as the `purge` and `property` subcommands pre-installed.
 
 > **Note**: When setting up your `.edgerc`, the `purge` subcommand defaults to the `ccu` credentials section, while the `property` subcommand uses the `papi` section. These can be changed using the `--section` flag.
 
 ### Compiling from Source
 
 If you want to compile it from source, you will need Go 1.7 or later, and the [Glide](https://glide.sh) package manager installed:
 
-1. Fetch the package:
```
Comment (dshafik) [inline README.md]: The double spaces here are required for markdown to break to a newline. I assume this is an Editor trimming extraneous trailing whitespace, but in this case it's actually necessary.

File: README.md [hunk 2]
```diff
@@ -59,21 +59,21 @@ If you use (or want to use) [docker](http://docker.com), you can get a fully ins
...
 
 If you want to compile it from source, you will need Go 1.7 or later, and the [Glide](https://glide.sh) package manager installed:
 
-1. Fetch the package:  
-  `go get github.com/akamai/cli`
+1. Fetch the package:
+  `go get -d github.com/akamai/cli`
 2. Change to the package directory:  
   `cd $GOPATH/src/github.com/akamai/cli`
-3. Install dependencies using Glide:
```
Comment (dshafik) [inline README.md]: Same here

File: README.md [hunk 3]
```diff
@@ -59,21 +59,21 @@ If you use (or want to use) [docker](http://docker.com), you can get a fully ins
...
-1. Fetch the package:  
-  `go get github.com/akamai/cli`
+1. Fetch the package:
+  `go get -d github.com/akamai/cli`
 2. Change to the package directory:  
   `cd $GOPATH/src/github.com/akamai/cli`
-3. Install dependencies using Glide:  
+3. Install dependencies using Glide:
   `glide install`
-4. Compile the binary:
```
Comment (dshafik) [inline README.md]: And here

File: README.md (hunk 1 above)
Comment (wpsmithtwc) [inline README.md]: In all my previews on VS Code, Atom and Jetbrains, they all render correctly??

File: README.md (hunk 2 above)
Comment (wpsmithtwc) [inline README.md]: In all my previews on VS Code, Atom and Jetbrains, they all render correctly??

File: README.md (hunk 3 above)
Comment (wpsmithtwc) [inline README.md]: In all my previews on VS Code, Atom and Jetbrains, they all render correctly??

**Pull request closed without merging**